### ISSUE TITLE
Zero shot embeddings

### ISSUE THREAD

**Issue by BaptistePodvin** [Project-MONAI/VISTA#63]
Hi, thanks for your work!
I was wondering how you trained your network to identify the zero-shot embeddings.
Are they only used during fine-tuning, or do you use pseudo-labels or supervoxel labels as training samples?

**Comment by heyufan1995** [comment]
Hi:
The zeroshot embedding is only used during interactive branch training. For interactive branch training, we use supervoxels to augment  ground truth manual label or pseudo labels from totalseg (https://github.com/Project-MONAI/VISTA/blob/8bb7572d163373ed73d136d2c0eacb152f79664c/vista3d/scripts/train.py#L583).  If it's supervoxel augmented, we treat it as zeroshot and will add zeroshot embedding.
